manifests/qpid_client.pp

```diff
@@ -39,16 +40,31 @@
 
   if $deploy {
 
+    file { $certs::pulp_pki_dir:
+      ensure => directory,
+      owner  => 'root',
+      group  => 'apache',
+      mode   => '0640',
+    }
+
+    file { "${certs::pulp_pki_dir}/qpid":
```

Comment:
Not 100% happy with the assumption that `$qpid_client_cert` and `$qpid_client_ca_cert` are not changed. Should we use a flat directory and name the files `qpid-client.crt` and `qpid-ca.crt`?

Reply:
We can go that route. I based this on the defaults and documentation from `pulp` itself:

https://github.com/pulp/pulp/blob/2.18-release/server/etc/pulp/server.conf#L309-L317